Tokens/dollartohousetoken/workflows/src/main/java/net/corda/examples/dollartohousetoken/flows/GiftTo.java

```diff
@@ -0,0 +1,68 @@
+package net.corda.examples.dollartohousetoken.flows;
```

Comment:
does it relate to 4.6 ungrade?

Reply:
not really, but it is a new flow to test Token 1.2